src/main/java/com/ss/lms/entity/Author.java

```diff
@@ -1,11 +1,42 @@
 package com.ss.lms.entity;
 
+import java.io.Serializable;
 import java.util.List;
 
-public class Author {
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Entity
+@Table(name = "tbl_author")
+public class Author implements Serializable {
+
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = -5362668537632529686L;
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private Integer authorId;
+
+	@Column(name = "authorName")
 	private String authorName;
-	//private List<Book> books; // many to many
+
+	@JsonIgnore
+	@ManyToMany(fetch = FetchType.EAGER)
+	@JoinTable(name = "tbl_book_authors", joinColumns = @JoinColumn(name = "authorId"), inverseJoinColumns = @JoinColumn(name = "bookId"))
```

Comment:
Also having it mapped in Authors could cause some problems down the line. 
If my understanding is correct you are currently defining Authors as the owner of the relationship. So making changes to the relational table it'll  happen from the author side.

I know in the current context you have it now it really doesnt matter because I dont think your updating the relationships but I think it may cause problems later.

Reply:
I just looked at your Books entity. why arent you using mappedBy = "authors"